db/schema.rb

```diff
@@ -124,6 +124,7 @@
     t.boolean "districtwide_access", default: false, null: false
     t.boolean "can_set_districtwide_access", default: false, null: false
     t.text "student_searchbar_json"
+    t.boolean "is_counselor", default: false
```

Comment:
Since this will default to false for all educators, we can turn this on incrementally, first for user testing with Les and then opening it up to all Counselor Educators.